features/remotepollerng/daemon/src/test/java/org/opennms/netmgt/remotepollerng/RemotePollerdIT.java

```diff
@@ -208,9 +208,10 @@ private void changePollingPackages(final String locationName, final String ... p
     }
 
     private void sendPollingPackageAssociationChanged(final String locationName) {
-        final EventBuilder ebldr = new EventBuilder(EventConstants.POLLER_PACKAGE_LOCATION_ASSOCIATION_CHANGED_EVENT_UEI, "test");
-        ebldr.addParam(EventConstants.PARM_LOCATION, locationName);
-        this.eventIpcManager.sendNow(ebldr.getEvent());
+        // TODO Patrick: do we need to send a different event?
```

Comment:
see above, the behavior for the partial reschedule of jobs for an application should be tested here.